canonical-web-and-design/microk8s.io
# Convert to flask and add docs

## Done
- Converted the site to flask
- But instead of installing template-finder I just switched the docs to the discourse all in this PR.

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8027/
- Check that the site looking the same as the live https://microk8s.io
- Go to docs
- Check they render nicely from https://discourse.ubuntu.com/t/introduction-to-microk8s/15536
